packages/zowe-explorer-api/src/profiles/ZoweExplorerApi.ts

```diff
@@ -435,13 +435,24 @@ export namespace ZoweExplorerApi {
         /**
          * Issues a TSO Command and returns a TsoSend API response.
          *
+         * @deprecated Use issueTsoCommandWithParms
          * @param {string} command
          * @param {string} acctNum
          * @returns {zowe.IIssueResponse>}
          * @memberof ICommand
          */
         issueTsoCommand?(command: string, acctNum?: string): Promise<zowe.IIssueResponse>;
 
+        /**
+         * Issues a TSO Command and returns a TsoSend API response.
+         *
+         * @param {string} command
+         * @param {zowe.IStartTsoParms} parms
+         * @returns {zowe.IIssueResponse>}
+         * @memberof ICommand
+         */
+        issueTsoCommandWithParms?(command: string, parms: zowe.IStartTsoParms): Promise<zowe.IIssueResponse>;
```

Comment:
parms should include the `?`, similar to the old API method for extenders that don't require this parameter.
`parms?: zowe.IStartTsoParms`